src/views/schedule.jade

```diff
@@ -1,4 +1,152 @@
 extends layout
 
 block content
-	h1(style="text-align:center;") Coming Soon
+	table
+		tr
+			th Home
+			th Away
+			th Date
+			th Time
+			th Result
+		tr
+			td.rangers
+				img(src = "/favicon-32x32.png")
+				span  Minnesota Rangers
+			td
+				img(src = "/broncos-32x32.png")
```

Comment:
Can you put the other team icons into a subfolder in images. images/otherLogos/  or something like that. I'd like to keep the root folder clean of non-ranger stuff if possible